packages/flutter_tools/test/web.shard/output_web_test.dart

```diff
@@ -77,5 +77,19 @@ void main() {
     await start();
     await sendEvent(<String, Object>{'type': 'DevtoolsEvent'});
     await warning;
-  });
+  }, skip: true); // Skipping for 'https://github.com/dart-lang/webdev/issues/1562'
```

Comment:
Did you intend to skip this test? (it wasn't skipped in the cherrypicked commit; however, I'm not sure if this test is also failing on this branch)

Reply:
Not intentionally, but I tried to handle the CP conflicts. I'll remove the skip then.